server/views/pages/manage-schedules/view-activity.njk

```diff
@@ -16,26 +17,78 @@
         <div class="govuk-grid-column-two-thirds-from-desktop">
             {{ govukSummaryList({
                 rows: [
+                    {
+                        key: {
+                            text: "Name"
+                        },
+                        value: {
+                            text: activity.summary
+                        },
+                        actions: {
+                            items: [
+                                {
```

Comment:
I believe we usually add `.govuk-link--no-visited-state` to changes links.

Reply:
Added